runtime/doc/vim_diff.txt

```diff
@@ -147,6 +146,15 @@ Highlight groups:
   |hl-TermCursorNC|
   |hl-Whitespace| highlights 'listchars' whitespace
 
+UI:
+		*E5408* *E5409* *g:Nvim_color_expr* *g:Nvim_color_cmdline*
+  UI now supports command-line coloring. Officially only |input()| and 
+  |inputdialog()| may be colored, temporary for testing purposes expressions 
+  (e.g. |i_CTRL-R_=|) and regular command-line (|:|) are colored by callbacks 
+  defined in `g:Nvim_color_expr` and `g:Nvim_color_cmdline` respectively. 
```

Comment:
@justinmk I can’t have leading underscores, funcref variable names must start with a *capital*. Underscore is not one.